infoschema/cluster.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package infoschema
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/parser/mysql"
+	"github.com/pingcap/tidb/domain/infosync"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/table"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util"
+)
+
+// Cluster table list, attention:
+// 1. the table name should be upper case.
+// 2. clusterTableName should equal to "TIDB_CLUSTER_" + memTableTableName.
```

Comment:
I think the prefix `CLUSTER_` is enough.

Reply:
em... The name was compatible with others,  such as `TIDB_CLUSTER_INFO`, `TIDB_CLUSTER_CONFIG`, `TIDB_CLUSTER_LOAD`...

Reply:
The TiDB prefix indicates this table is only exists in TiDB